### PR TITLE
Adding initial project files

### PR DESCRIPTION
**Pull request by rmcmurray-ch**

### CODE REVIEW

**Review by HarryBosworth — APPROVED** [review]
LGTM

**Review by adeharrison-ch — CHANGES_REQUESTED** [review]
one very, very minor question (which probably depends on where the structure was copied from), but should "api" be in there?

"src/main/java/uk/gov/companieshouse/filetransferservice/api/Controller/FileTransferController.java

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven-surefire-plugin.version}</version>
+                <configuration>
+                    <argLine>-Dfile.encoding=UTF-8</argLine>
+                    <argLine>${surefireArgLine}</argLine>
+                    <excludes>
+                        <exclude>**/uk.gov.companieshouse.account.validator.Runner.java</exclude>
```
Comment (HarryBosworth) [inline pom.xml:236]: Should this be changed to `file-transfer-service`?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <argLine>-Dfile.encoding=UTF-8</argLine>
+                    <argLine>${failsafeArgLine}</argLine>
+                    <classesDirectory>${project.build.outputDirectory}</classesDirectory>
+                    <skipITs>${skip.integration.tests}</skipITs>
+                    <includes>
+                        <include>**/uk.gov.companieshouse.account.validator.Runner.java</include>
```
Comment (HarryBosworth) [inline pom.xml:306]: Should this be changed to `file-transfer-service`?

**Review by HarryBosworth — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (rmcmurray-ch) [reply pom.xml:236]: good catch thanks, missed that one in copying over from account-validator-api - will update

File: pom.xml (hunk 2 above)
Comment (rmcmurray-ch) [reply pom.xml]: Updated commit

**Review by adeharrison-ch — CHANGES_REQUESTED** [review]
ah just noticed "Controller" is usually lower case

**Pull request merged**